### DISCUSSION TITLE
Flag support like in normal UTAU

### DISCUSSION THREAD

**Discussion by AnAndroNerd** [stakira/OpenUtau#764] [Ideas]
So you know how in UTAU you have "Rendering Options" where you can put flags in, can this feature be added to OpenUtau? This is my last big problem with OpenUtau because a lot of voicebanks sound way better by using flags in a way you can't in do in OpenUtau.

**Comment by oxygen-dioxide** [comment]
> So you know how in UTAU you have "Rendering Options" where you can put flags in, can this feature be added to OpenUtau? This is my last big problem with OpenUtau because a lot of voicebanks sound way better by using flags in a way you can't in do in OpenUtau.

You can add it as an options expression. for example, here is how to add "Mt10Mb10e" as the default flag:
<img width="452" alt="image" src="https://github.com/stakira/OpenUtau/assets/54425948/6a788e1c-5cd3-42e4-ad92-a06c6be29959">

then you can use it in your song.
<img width="711" alt="image" src="https://github.com/stakira/OpenUtau/assets/54425948/e9df8410-da9d-4076-b317-d68dbc01badf">
